mlflow/server/handlers.py

```diff
@@ -530,7 +530,7 @@ def validate_path_is_safe(path):
     if (
         any((s in path) for s in _OS_ALT_SEPS)
         or ".." in path.split(posixpath.sep)
-        or posixpath.isabs(path)
+        or pathlib.Path(path).is_absolute()
```

Comment:
btw do we need to support windows paths?

Reply:
or is the challenge posix path detection?

Reply:
Yes, on windows machine 
<img width="238" alt="image" src="https://github.com/mlflow/mlflow/assets/82044803/5a3f5607-4406-4579-a635-23e0373b90e7">